content/_changelogs/10.8.0.md

```diff
@@ -0,0 +1,106 @@
+## 10.7.0
+
+_Released 9/13/2022_
+
+**Features:**
+
+- You can now run tests in Webkit, Safari's browser engine, by enabled the
+  [`experimentalWebKitSupport`](/guides/references/experiments) experiment and
+  installing
+  [`playwright-webkit`](https://www.npmjs.com/package/playwright-webkit). For
+  more details, read
+  [our blog post](https://www.cypress.io/blog/2022/09/13/cypress-10-8-experimental-run-tests-in-webkit).
+  Addressed [#17336](https://github.com/cypress-io/cypress/issues/6422).
+
+**Performance:**
+
+- Several minor changes were made to reduce Cypress's memory footprint in the
+  browser in an attempt to improve test times and reduce the likelihood of
+  crashing the launched browser under test. Addressed in
+  [#23708](https://github.com/cypress-io/cypress/pull/23708),
+  [#23729](https://github.com/cypress-io/cypress/pull/23729),
+  [#23737](https://github.com/cypress-io/cypress/pull/23737), and
+  [#23738](https://github.com/cypress-io/cypress/pull/23738).
+
+**Bugfixes:**
+
+- Corrected the spec results that are published to the Dashboard to correctly
+  show whether a video of the spec was uploaded to review. Before this change,
+  the Dashboard would shows the error message `The media could not be loaded`
+  when a passing test's video was not uploaded because `videoUploadOnPasses` was
+  `false`. Fixes [#2939](https://github.com/cypress-io/cypress/issues/2939).
+- Fixed an issue where the stack traces in `run` mode did not match the stack
+  traces shown in `open` mode. Now, In the terminal, users will see the source
+  map line and column number in the error stack trace. Fixes
+  [#7715](https://github.com/cypress-io/cypress/issues/7715).
+- Fix a bug where projects using Node.js 16.17+ and 18.6+ with ES Modules and
+  TypeScript were not working with Cypress. Fixes
+  [#22795](https://github.com/cypress-io/cypress/issues/22795),
+  [#23393](https://github.com/cypress-io/cypress/issues/23393) and
+  [#23552](https://github.com/cypress-io/cypress/issues/23552).
+- When users on Windows use the spec list search, if their search term contains
+  any instances of `/`, we now replaced with `\` to correctly match the Windows
+  path. Fixes [#23426](https://github.com/cypress-io/cypress/issues/23426).
```

Comment:
```suggestion
- When searching for specs we now normalize OS-specific path separators so that Windows users can use back- and forward-slashes. Fixes [#23426](https://github.com/cypress-io/cypress/issues/23426).
```